imagepicker/src/main/java/com/esafirm/imagepicker/view/SnackBarView.kt

```diff
@@ -0,0 +1,80 @@
+package com.esafirm.imagepicker.view
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.view.animation.Interpolator
+import android.widget.Button
+import android.widget.RelativeLayout
+import android.widget.TextView
+import androidx.annotation.StringRes
+import androidx.core.view.ViewCompat
+import androidx.interpolator.view.animation.FastOutLinearInInterpolator
+import com.esafirm.imagepicker.R
+import kotlinx.android.synthetic.main.ef_imagepikcer_snackbar.view.*
+
+class SnackBarView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+) : RelativeLayout(context, attrs, defStyle) {
+
+    private var txtCaption: TextView? = null
+    private var btnAction: Button? = null
```

Comment:
maybe use lateinit or `get() =` in property

Reply:
Because it's always there, we just don't want to invoke it in edit mode (layout editor)